src/loopback-model-definition.json

```diff
@@ -162,7 +162,7 @@
           },
           "type" : {
             "type" : "string",
-            "enum" : ["hasOne","hasMany","belongsTo","hasAndBelongsToMany"]
+            "enum" : ["hasOne","hasMany","belongsTo","hasAndBelongsToMany", "embedsOne", "embedsMany"]
```

Comment:
```js
["hasOne","hasMany","belongsTo","hasAndBelongsToMany", "embedsOne", "embedsMany", "referencesMany"]
```
https://loopback.io/doc/en/lb3/Embedded-models-and-relations.html#referencesmany